ds-controller-board/driver/driver.go

```diff
@@ -260,3 +279,24 @@ func (drv *ControllerBoardDriver) RemoveDevice(deviceName string, protocols map[
 func (drv *ControllerBoardDriver) Stop(force bool) error {
 	return nil
 }
+
+func (drv *ControllerBoardDriver) updateWritableConfig(rawWritableConfig interface{}) {
+	updated, ok := rawWritableConfig.(*device.CustomConfig)
+	if !ok {
+		drv.lc.Error("unable to update writable custom config: Can not cast raw config to type 'CustomConfig'")
```

Comment:
VID and PID need to be move into the Device Protocol. So this is not needed.
If we allows writable you have to consider how the value is used and if there is special processing needed when the value changes. IMO, these settings are not going to need to change at run time to justify this complexity.

Reply:
Correct, VID, PID, and simulated will require a service reboot.